docs/application/native/guides/text-input/autofill.md

```diff
@@ -155,7 +139,7 @@ To get the presence of Autofill data and Autofill authentication information:
     autofill_auth_info_set_received_cb(ah, auth_info_received_cb, NULL);
     ```
 
-2. To include several Autofill data, create autofill data for each input field and view information:
+2. To include several autofill data, create autofill data for each input field and view information:
```

Comment:
Create autofill data for each input field and view information to include several autofill data:

Reply:
fixed